scriptworker_client/src/scriptworker_client/github_client.py

```diff
@@ -87,7 +87,7 @@ async def _execute():
 
         await retry_async(_execute, attempts=3, retry_exceptions=(TransportQueryError,), sleeptime_kwargs={"delay_factor": 0})
 
-    async def get_files(self, files: Union[str, List[str]], branch: Optional[str] = None) -> Dict[str, str]:
+    async def get_files(self, files: Union[str, List[str]], branch: Optional[str] = None) -> Dict[str, Union[str, None]]:
```

Comment:
nit: if I'm not mistaken, we should be using Python 3.11 for scriptworkers now. Which means that `str | None` is valid syntax. Though I guess it's best to be consistent and I see `Union` used elsewhere.

Reply:
iscript (and therefore also scriptworker_client) is still on 3.8 iirc.